packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/vmware-cloud-director/controller.js

```diff
@@ -20,6 +28,12 @@ export default class {
             ? 'dedicatedCloud_vmware_cloud_director_migration'
             : 'dedicatedCloud_vmware_cloud_director_validate_migration',
         );
+
+        if (this.vcdMigrationState.isDone) {
+          this.atInternet.trackPage({
+            name: `${this.trackingPrefix}vmware::vmware::banner-info::migrate_to_managed_vcd_success`,
+          });
+        }
```

Comment:
Not really sure it's well located. Migration will be done asynchronously and in a different timeline.
What's the condition to enable that tracking value ?